openedx_ledger/models.py

```diff
@@ -353,6 +363,16 @@ class Meta:
             "The globally unique content identifier.  Joinable with ContentMetadata.content_key in enterprise-catalog."
         )
     )
+    content_title = models.CharField(
+        max_length=255,
```

Comment:
This may be too short for content titles.

Reply:
Looks like discovery uses 255 - https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/course_metadata/models.py#L1319

make it bigger here anyway?